### PR TITLE
Add NewUsageNotification class for Recurly webhook

### PR DESCRIPTION
**Pull request by gregthebusker**

### CODE REVIEW

**Comment by bhelx** [comment]
Thanks for contributing @gregthebusker !

I'll take a look at reviewing and merging this after it goes green.

**Comment by gregthebusker** [comment]
@bhelx Can you rerun the tests(I don't think I have permission to)?  It looks like the test failed from an unrelated network timeout.

**Comment by bhelx** [comment]
Seems like it was just some hiccups in travis. :+1:

**Comment by gregthebusker** [comment]
@bhelx The tests passed

File: lib/recurly/webhook/new_usage_notification.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module Recurly
+  module Webhook
+    class NewUsageNotification < AccountNotification
```
[the remark this answers is not in the document]
Comment (gregthebusker) [reply lib/recurly/webhook/new_usage_notification.rb]: @bhelx done

**Comment by bhelx** [comment]
Thanks for your commits and your patience @gregthebusker!

**Pull request merged**